assignment/index.html

```diff
@@ -64,6 +64,16 @@
 
       var jsonToCsv = function(json) { console.log(json); };
 
+      var csvHC = [];
+      var header = Object.keys(healthCenters[0]);
+      csvHC.push(header)
+      for (var i = 0; i < healthCenters.length; i = i+1) {
+      csvHC.push(Object.values(healthCenters[i]))
+      };
```

Comment:
This does indeed produce the intended output.

But it does not actually log it to the console.

Can you modify the jsonToCsv function such that it executes these commands when then function is called below.